tests/test_apexrest.py

```diff
@@ -0,0 +1,70 @@
+import testutil
+import responses
+
+@responses.activate
+def test_apexrest_get_request():
+	testutil.add_response("login_response_200")
+	testutil.add_response("apexrest_get_response_200")
+	testutil.add_response("api_version_response_200")
+
+	client = testutil.get_client()
+	apexrest_result = client.apexrest(
+		action='foo',
+		http_method='GET',
+		request_params={"foo": 0}
```

Comment:
Hi. The idea of request_params is that in this case the following URL would be generated: "https://eu11.salesforce.com/services/apexrest/foo?foo=0". Just double checking if you agree with the intention of request_params, since the test cases are not checking for that explicitly...